packages/@misk/core/src/features/Navbar/MiskContainer.tsx

```diff
@@ -37,6 +37,11 @@ export const testServiceMetadataUrl =
 export const MiskNavbarContainer = (
   props: IMiskNavbarContainerProps & INavbarProps
 ) => {
+  let alignment: Alignment = Alignment.LEFT
```

Comment:
This hopefully can get removed with the inline default I explained below.

Reply:
On second thought, I think that `Alignment.Left` is the default value for that props so we can just eliminate this block entirely.

https://blueprintjs.com/docs/#core/components/navbar